### PR TITLE
Add back guest attributes with fix for metadata shutdown test

### PR DESCRIPTION
**Pull request by koln67**
Previously, guest attributes were leading to the metadata shutdown script test failing due to reboot behaviors. This should fix the issue.

### CODE REVIEW

**Comment by dorileo** [comment]
I'm holding this PR until we have releases done this week.

**Comment by koln67** [comment]
It seems that the issue was not the interval, but that the guest attribute stayed active on reboot.

If a test needed to run after the reboot finished, it could not do that, as the daisy workflow would immediately end. This is because the guest attribute was already set on the previous boot.

File: imagetest/cmd/wrapper/main.go [hunk 1]
```diff
@@ -23,12 +24,21 @@ func main() {
...
+		if err := utils.QueryMetadataGuestAttribute(ctx, utils.GuestAttributeTestNamespace, utils.GuestAttributeTestKey, http.MethodPut); err != nil {
+			log.Printf("failed to put test completed key in guest attribute namespace")
+		}
+		log.Printf("successfully placed guest attribute for test completion")
 		for f := 0; f < 5; f++ {
 			log.Printf("FINISHED-TEST")
 			time.Sleep(1 * time.Second)
 		}
-	}()
+		time.Sleep(10 * time.Second)
```
Comment (dorileo) [inline imagetest/cmd/wrapper/main.go:36]: I understand why this is needed here, however I believe the actual fix should go in the metadata script runner - we can further discuss it offline.

Comment (dorileo) [reply imagetest/cmd/wrapper/main.go:36]: We can get rid of it when we have a fix landed to metadata script runner.

**Pull request merged**